code/datums/components/crafting/recipes.dm

```diff
@@ -1141,25 +1141,27 @@
 	reqs = list(/obj/item/aicard = 1,
 					/obj/item/food/grown/potato = 1,
 					/obj/item/stack/cable_coil = 5)
+	parts = list(/obj/item/aicard = 1)
```

Comment:
Is it necessary for it to be a part too?

Reply:
Being a part is what makes it get moved into the contents of the crafted item. If it wasn't a part then it'd just get deleted along with the AI inside it